ModularTegustation/fishing/code/rod_components/fishing_hooks.dm

```diff
@@ -18,7 +18,7 @@
 	name = "bone hook"
 	desc = "a simple hook carved from sharpened bone"
 	icon_state = "bone"
-	fishing_value = 200
+	fishing_value = 0.1
```

Comment:
This is now a downgrade (base is 0.2, reportedly)

Reply:
changed it to 0.3